JeffShepherd/Rancid-Tomatillos
# Feature/display movies

## What does this PR do?
- Adds basic structure to UI
- Displays mock movie data to UI

## How should it be tested in the UI?
- cd into `Rancid-Tomatillos` repo, run `npm start`
- All objects should appear on the dashboard on load
